core/domain/story_domain.py

```diff
@@ -968,3 +968,15 @@ def to_dict(self):
             'story_model_last_updated': utils.get_time_in_millisecs(
                 self.story_model_last_updated)
         }
+
+    def to_human_readable_dict(self):
```

Comment:
I really don't want to have to keep saying this again and again, but when you add a new method, you must add tests for it. Please bear this in mind in the future.

Reply:
Added, sorry for the repeated comment.